get-started/07_multi_container.md

```diff
@@ -0,0 +1,265 @@
+    (your IP address will most likely have a different value). While `mysql` isn't normally a valid hostname,
+    Docker was able to resolve it to the IP address of the container that had that network alias (remember the
+    `--network-alias` flag we used earlier?).
+
+    What this means is... our app only simply needs to connect to a host named `mysql` and it'll talk to the
+    database! It doesn't get much simpler than that!
+
+
+## Running our App with MySQL
+
+The todo app supports the setting of a few environment variables to specify MySQL connection settings. They are:
+
+- `MYSQL_HOST` - the hostname for the running MySQL server
+- `MYSQL_USER` - the username to use for the connection
+- `MYSQL_PASSWORD` - the password to use for the connection
+- `MYSQL_DB` - the database to use once connected
+
+>**warning** Setting Connection Settings via Env Vars
+>While using env vars to set connection settings is generally ok for development, it is **HIGHLY DISCOURAGED**
+>when running applications in production. Diogo Monica, the former lead of security at Docker, 
+>[wrote a fantastic blog post](https://diogomonica.com/2017/03/27/why-you-shouldnt-use-env-variables-for-secret-data/)
+>explaining why. 
+>  
+>A more secure mechanism is to use the secret support provided by your container orchestration framework. In most cases,
+>these secrets are mounted as files in the running container. You'll see many apps (including the MySQL image and the todo app)
+>also support env vars with a `_FILE` suffix to point to a file containing the variable. 
+>   
+>As an example, setting the `MYSQL_PASSWORD_FILE` var will cause the app to use the contents of the referenced file 
+>as the connection password. Docker doesn't do anything to support these env vars. Your app will need to know to look for
+>the variable and get the file contents.
+
+
+With all of that explained, let's start our dev-ready container!
+
+1. We'll specify each of the environment variables above, as well as connect the container to our app network.
+
+    ```bash hl_lines="3 4 5 6 7"
+    docker run -dp 3000:3000 \
+      -w /app -v "$(pwd):/app" \
+      --network todo-app \
+      -e MYSQL_HOST=mysql \
+      -e MYSQL_USER=root \
+      -e MYSQL_PASSWORD=secret \
+      -e MYSQL_DB=todos \
+      node:12-alpine \
+      sh -c "yarn install && yarn run dev"
+    ```
+
+    If you are using PowerShell then use this command.
+
+    ```powershell hl_lines="3 4 5 6 7"
+    docker run -dp 3000:3000 `
+      -w /app -v "$(pwd):/app" `
+      --network todo-app `
+      -e MYSQL_HOST=mysql `
+      -e MYSQL_USER=root `
+      -e MYSQL_PASSWORD=secret `
+      -e MYSQL_DB=todos `
+      node:12-alpine `
+      sh -c "yarn install && yarn run dev"
+    ```
+
+1. If we look at the logs for the container (`docker logs <container-id>`), we should see a message indicating it's
+   using the mysql database.
+
+    ```plaintext hl_lines="7"
```

Comment:
Looks like formatting is off - https://deploy-preview-11838--docsdocker.netlify.app/get-started/07_multi_container/#running-our-app-with-mysql

Reply:
Fixed